packages/dm-core-plugins/src/role_filter/RoleFilterPlugin.tsx

```diff
@@ -87,7 +75,7 @@ export const RoleFilterPlugin = (props: IUIPlugin): JSX.Element => {
             <Icon name="thumbs_down" />
           </Banner.Icon>
           <Banner.Message>
-            {`No views found, since you currently have roles [${User.roles}]. Please switch to one of these roles: [${allowedRoles}]`}
+            {`No views found, since you currently have role [${selectedRole}]. Please switch to one of these roles: [${allowedRoles}]`}
```

Comment:
I understand what you mean. Right now you can define a view without any roles, which will show for users without a role (or with a wrong role). Like this you are not forced to show anything, which is nice imo. If we fallback to e.g. the yaml plugin, now we have introduced a dependency, which sounds like something we want to avoid. I am still not super confident on the plans for the future, but it would be a problem if say you dont want to install the yaml plugin (if that is a use case). Any thoughts on this @eoaksnes @henrikbossart @pbullhove ?

Reply:
Maybe the fallback can be an optional parameter? I can imagine cases where an application would use a fallback in case of lacking access permissions, but I can also see cases where an application might choose to just show an "you don't have access to this resource/view/entity" disclaimer. 

I do agree about it being hard to say anything about the future needs of the framework though 😅

Reply:
Agreed we keep the flexibility of optional fallback in the config for RolerFilterPlugin :+1: